packages/manager/modules/overthebox/src/overthebox/autoconfigure/dhcp/template.html

```diff
@@ -0,0 +1,284 @@
+                            class="oui-input"
+                            name="priority"
+                            type="number"
+                            required
+                            ng-model="$ctrl.config.priority"
+                        />
+                    </oui-field>
+                    <oui-button
+                        class="ml-2 mb-2"
+                        variant="primary"
+                        type="submit"
+                        on-click="$ctrl.createConfig()"
+                    >
+                        <span
+                            data-translate="overTheBox_autoconfigure_submit"
+                        ></span>
+                    </oui-button>
+                    <oui-button
+                        class="ml-2 mb-2"
+                        variant="primary"
+                        on-click="$ctrl.cancelCreateConfig()"
+                    >
+                        <span
+                            data-translate="overTheBox_autoconfigure_cancel"
+                        ></span>
+                    </oui-button>
+                </div>
+            </form>
+        </div>
+        <oui-button
+            variant="primary"
+            data-on-click="$ctrl.addConfig()"
+            class="mb-2 d-flex justify-content-center"
+            data-ng-if="$ctrl.displayAddConfigButton"
+        >
+            <span
+                data-translate="overTheBox_autoconfigure_dhcp_config_add"
+            ></span>
+        </oui-button>
+        <oui-datagrid
+            page-size="5"
+            rows="$ctrl.configList"
+            data-ng-if="$ctrl.configList.length > 0"
+        >
+            <oui-datagrid-column
+                title="'overTheBox_autoconfigure_dhcp_interface' | translate"
+                property="configuration.interface"
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title="'overTheBox_autoconfigure_dhcp_offset' | translate"
+                property="configuration.offset"
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title="'overTheBox_autoconfigure_dhcp_pool_size' | translate"
+                property="configuration.poolSize"
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column
+                title="'overTheBox_autoconfigure_dhcp_lease_duration' | translate"
+                property="configuration.leaseDuration"
+            ></oui-datagrid-column>
+            <oui-datagrid-column
+                title="'overTheBox_autoconfigure_priority' | translate"
+                property="priority"
+            >
+            </oui-datagrid-column>
+            <oui-datagrid-column>
+                <oui-button
+                    variant="primary"
+                    size="s"
+                    on-click="$ctrl.removeConfig($row)"
+                >
+                    <span class="oui-icon oui-icon-trash"></span>
```

Comment:
```suggestion
                    <span class="oui-icon oui-icon-trash" aria-hidden="true"></span>
```
Also we should add a `sr-only` description